hrpe/submit/create_submission.py

```diff
@@ -0,0 +1,83 @@
+from zipfile import ZipFile
+from glob import glob
+from datetime import datetime
+from calendar import monthrange
+from os import remove
+import pandas as pd
+
+from hrpe.data.load import filter_data_by_time
+
+
+def create_submissions_file(
+    data: pd.DataFrame, month: str, userinitials: str, year=2021
+):
+    """[Create a submission file for the HRPE]
+
+    Args:
+        data (pd.DataFrame): data must have the columns 'time', 'value_max', 'value_min'
+        month ([type]): text desc of a month i.e 'august'
+        userinitials ([type]): aa, ml, cl
+        year (int, optional): [description]. Defaults to 2021.
+
+    Example:
+        from hrpe.data.load import load_maxmin_data
+        data = load_maxmin_data('staplegrove')
+        create_submissions_file(data, 'july', 'aa')
+
+    Returns:
+        [type]: [description]
+    """
+    userinitials = userinitials.lower()
+    assert userinitials in ["aa", "ml", "cl"]
+
+    data_filt = _validate_submissions_data(data, month, year)
+    csv_path = _create_submission_csv(data_filt, userinitials)
+    zippath = _zip_submissions_file(csv_path)
+    return zippath
+
+
+def _validate_submissions_data(data, month, year):
+
+    assert all(
+        ["time", "value_max", "value_min"] == data.columns
```

Comment:
not sure if this works as a method for matching cols as if there are extra cols in data it will fail?

Reply:
Yes, we could replace it such that it only selects those columns, but that's all we can submit